app/views/accommodations/show.html.erb

```diff
@@ -1,10 +1,6 @@
-<div class="row">
-  <h3>Editing accommodation '<%= @accommodation.label %>' in building '<%= @accommodation.building.label %>'</h3>
-</div>
+<h3>Editing accommodation '<%= @accommodation.label %>'</h3>
```

Comment:
removed the second part because accommodation.label already includes building.label